chapters/statements.tex

```diff
@@ -392,7 +392,7 @@ \subsection{When-Statements}\label{when-statements}
 Merging the \lstinline!when!-statements can lead to less efficient code and different models with different behavior depending on the order of the assignment to \lstinline!y1! and \lstinline!y3! in the algorithm.
 \end{example}
 
-\subsubsection{Restrictions on Where a When-statement may occur}\label{restrictions-on-where-a-when-statement-may-occur}
+\subsubsection{Where a When-statement May Occur}\label{restrictions-on-where-a-when-statement-may-occur}\label{where-a-when-statement-may-occur}
```

Comment:
Now we have For-Statements, If-Statements, even When-Statements as the subsection name. But here and in the changes below (lines 419, 434) they became When-statements.